atd-etl/reassociate_missing_locations/scripts/atd_vzd_reassociate_wrong_locations.py

```diff
@@ -0,0 +1,80 @@
+import json
+import os
+import requests
+import time
+
+
+def make_update() -> dict:
+    """
+    Runs a SQL query against Hasura and tries to dissociate the location
+    for any CR3 crashes that fall in the main-lane polygon.
+    :return dict:
```

Comment:
I know that this isn't your code and it was copy/pasted, but I noticed that this has the same description as the one in `atd_vzd_dissociate_cr3_mainlanes.py` when I was going through and trying to refresh on what these scripts do.

Reply:
Yea thank you for pointing that out! I worked on this description some to make it clearer, I think.